src/autoscaler/eventgenerator/client/client_suite_test.go

```diff
@@ -10,7 +10,6 @@ import (
 )
 
 func TestClient(t *testing.T) {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Seems to be not needed, I have not discovered calls to `math.rand..*` in any of the tests within that directory.